esmgrids/util.py

```diff
@@ -46,39 +42,6 @@ def calc_area_of_polygons(clons, clats):
     return areas
 
 
-def is_git_repo():
-    """
-    Return True/False depending on whether or not the current directory is a git repo.
-    """
-
-    return subprocess.call(["git", "-C", ".", "status"], stderr=subprocess.STDOUT, stdout=open(os.devnull, "w")) == 0
-
-
-def git_info():
-    """
-    Return the git repo origin url, relative path to this file, and latest commit hash.
-    """
-
-    url = subprocess.check_output(["git", "remote", "get-url", "origin"]).decode("ascii").strip()
-    top_level_dir = subprocess.check_output(["git", "rev-parse", "--show-toplevel"]).decode("ascii").strip()
-    rel_path = os.path.relpath(__file__, top_level_dir)
-    hash = subprocess.check_output(["git", "rev-parse", "HEAD"]).decode("ascii").strip()
-
-    return url, rel_path, hash
-
-
-def create_nc(filename):
-    f = Dataset(filename, "w")
-
-    f.timeGenerated = f"{datetime.now()}"
-    f.created_by = f"{os.environ.get('USER')}"
-    if is_git_repo():
-        git_url, _, git_hash = git_info()
-        f.history = f"Created using commit {git_hash} of {git_url}"
```

Comment:
Don't we need this. Like its seems very optimistic to assume we will only run this from a released version of the package ? (And we don't have a good track record of getting version number right and unique! :))

Reply:
I think production versions of files should only be created from released versions. But we're using `setuptools_scm` for versioning so the version name will include the latest commit since a release